trigger/pom.xml

```diff
@@ -20,87 +20,107 @@
+    </properties>
+
+
     <dependencies>
         <dependency>
-            <groupId>za.co.absa</groupId>
-            <artifactId>abris_2.11</artifactId>
-            <version>${abris.version}</version>
+            <groupId>com.typesafe.slick</groupId>
+            <artifactId>slick_2.11</artifactId>
+            <version>3.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.11</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_${scala.compat.version}</artifactId>
+            <version>2.2.0</version>
         </dependency>
-
-        <!--Components from the project-->
         <dependency>
-            <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>reader</artifactId>
-            <version>${project.version}</version>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-launcher_${scala.compat.version}</artifactId>
+            <version>${spark.version}</version>
         </dependency>
         <dependency>
-            <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>transformer</artifactId>
-            <version>${project.version}</version>
+            <groupId>com.typesafe.play</groupId>
+            <artifactId>play-json_${scala.compat.version}</artifactId>
+            <version>2.7.3</version>
         </dependency>
         <dependency>
-            <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>writer</artifactId>
-            <version>${project.version}</version>
+            <groupId>com.typesafe.play</groupId>
+            <artifactId>play-ws-standalone-json_2.11</artifactId>
+            <version>2.0.3</version>
         </dependency>
         <dependency>
-            <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>manager</artifactId>
-            <version>${project.version}</version>
+            <groupId>com.typesafe.play</groupId>
+            <artifactId>play-ahc-ws-standalone_2.11</artifactId>
+            <version>2.0.2</version>
         </dependency>
         <dependency>
-            <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>driver</artifactId>
-            <version>${project.version}</version>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <version>2.0.0.RELEASE</version>
         </dependency>
         <dependency>
-            <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>shared</artifactId>
-            <version>${project.version}</version>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-scala_${scala.compat.version}</artifactId>
+            <version>2.9.8</version>
         </dependency>
     </dependencies>
```

Comment:
It might also be helpful to define a `<dependencyManagement>` section, specially if this module is expected to become a parent one.

Reply:
Yes, I will keep it in mind. Will be changed. Its not priority now.